Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/valueset/valuesets/TestCertificateValueSets.kt

```diff
@@ -1,18 +1,18 @@
 package de.rki.coronawarnapp.covidcertificate.valueset.valuesets
 
-import androidx.annotation.Keep
-import com.google.gson.annotations.SerializedName
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonProperty
 import java.util.Locale
 
-@Keep
 data class TestCertificateValueSets(
-    @SerializedName("languageCode") override val languageCode: Locale,
-    @SerializedName("tg") override val tg: DefaultValueSet,
-    @SerializedName("tt") val tt: DefaultValueSet, // Type of Test
-    @SerializedName("ma") val ma: DefaultValueSet, // RAT Test name and manufacturer
-    @SerializedName("tr") val tr: DefaultValueSet, // Test Result
+    @JsonProperty("languageCode") override val languageCode: Locale,
+    @JsonProperty("tg") override val tg: DefaultValueSet,
+    @JsonProperty("tt") val tt: DefaultValueSet, // Type of Test
+    @JsonProperty("ma") val ma: DefaultValueSet, // RAT Test name and manufacturer
+    @JsonProperty("tr") val tr: DefaultValueSet, // Test Result
```

Comment:
and here

Reply:
I changed ValueSetsStorage to use Jackson because it didn't used TypeAdapterFactory. This included ValueSetsContainer, VaccinationValueSets and TestCertificateValueSets which was working well. 
Nevertheless I realised that it may be better to use BaseJsonSerializer than the objectMapper directly, so I reverted this change now to have this refactored in the Jackson story